handlers/meshery_application_handler.go

```diff
@@ -167,6 +181,13 @@ func (h *Handler) GetMesheryApplicationsHandler(
 	fmt.Fprint(rw, string(resp))
 }
 
+// swagger:route DELETE /api/experimental/application/{id} ApplicationAPI idDeleteMesheryApplicationFile
+// Handle Delete for a Meshery Application File
+//
+// Deletes a meshery application file with ID: id
+// response:
+// 	200: IDParameterWrapper
```

Comment:
How can we attach a parameter as a response?

Reply:
Oh yes I am sorry but I think I should have used `noContentWrapper` right?

Reply:
Yeah, you can add that or leave it empty.